endpoints/openrtb2/auction.go

```diff
@@ -1317,23 +1322,34 @@ func getJsonSyntaxError(testJSON []byte) (bool, string) {
 	return false, ""
 }
 
-func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson []byte) ([]byte, map[string]exchange.ImpExtInfo, []error) {
+func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson []byte, impInfo []ImpData) ([]byte, map[string]exchange.ImpExtInfo, []error) {
 	// Parse the Stored Request IDs from the BidRequest and Imps.
 	storedBidRequestId, hasStoredBidRequest, err := getStoredRequestId(requestJson)
 	if err != nil {
 		return nil, nil, []error{err}
 	}
-	imps, impIds, idIndices, errs := parseImpInfo(requestJson)
-	if len(errs) > 0 {
-		return nil, nil, errs
-	}
 
 	// Fetch the Stored Request data
 	var storedReqIds []string
 	if hasStoredBidRequest {
 		storedReqIds = []string{storedBidRequestId}
 	}
-	storedRequests, storedImps, errs := deps.storedReqFetcher.FetchRequests(ctx, storedReqIds, impIds)
+
+	//impStoredReqIds list of all stored impression ids
+	impStoredReqIds := make([]string, 0)
+	impStoredReqIdsUnique := make(map[string]interface{})
```

Comment:
Consider renaming `impStoredReqIdsUnique` to `impStoredReqIdsUniqueTracker` so it's more clear the map is temporary and only used for deduplication.

Reply:
Renamed